src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,20 @@
+import { useParams } from 'react-router-dom';
+import { TabsTable } from '../TabsTable';
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const TabsPage = () => {
+  const { tabId = '0' } = useParams();
```

Comment:
let's not set any value at all, if it's not there - it's not there
```suggestion
  const { tabId} = useParams();
```